soundfile.py

```diff
@@ -273,9 +273,18 @@
         _libname = 'libsndfile' + _architecture()[0] + '.dll'
     else:
         raise
+
+    # hack for packaging tools like cx_Freeze, which
+    # compress all scripts into a zip file
+    # which causes __file__ to be inside this zip file
+
+    path = _os.path.dirname(_os.path.abspath(__file__))
```

Comment:
You should pre-fix `path` with an underscore, otherwise it pollutes the module namespace.